DeepLinkDemo/DeepLinkDemo/NSURLSessionBranch.m

```diff
@@ -7,6 +7,7 @@
 
 #import "NSURLSessionBranch.h"
```

Comment:
Swizzles are normally named like categories`NSURLSession+Branch.h". Not a blocking suggestion though.